app/src/main/res/values/style.xml

```diff
@@ -21,15 +21,47 @@
         <item name="android:layout_marginTop">@dimen/margin_top_edit_text_dialog_fragment</item>
     </style>
 
-    <style name="DialogFragmentSaveButtonStyle">
+    <style name="SaveButtonStyle">
         <item name="android:text">@string/button_save_dialog_fragment_parking_size</item>
         <item name="android:backgroundTint">@color/yellow</item>
+        <item name="android:layout_marginRight">@dimen/margin_right_small</item>
     </style>
 
-    <style name="DialogFragmentCancelButtonStyle">
+    <style name="CancelButtonStyle">
         <item name="android:text">@string/button_cancel_dialog_fragment_parking_size</item>
         <item name="android:backgroundTint">@color/yellow</item>
         <item name="android:layout_marginBottom">@dimen/margin_bottom_button_dialog_fragment_small</item>
     </style>
 
+    <style name="MainReservationButtonStyle">
+        <item name="android:text">@string/button_main_add_reservation</item>
+        <item name="android:backgroundTint">@color/yellow</item>
+        <item name="android:layout_marginBottom">@dimen/margin_bottom_button_dialog_fragment_small</item>
+    </style>
+
+    <style name="ActivityReservationTitle">
+        <item name="android:text">@string/title_activity_reservation</item>
+        <item name="android:textColor">@color/white</item>
+        <item name="android:layout_margin">@dimen/margin_top_title_activity_reservation</item>
+        <item name="android:textSize">@dimen/dimen_title_big</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="ButtonInitialDateActivityReservation">
+        <item name="android:text">@string/button_initial_date_time_picker_reservation</item>
+        <item name="android:backgroundTint">@color/yellow</item>
+    </style>
+
+    <style name="ButtonEndDateActivityReservation">
+        <item name="android:text">@string/button_end_date_time_picker_reservation</item>
+        <item name="android:backgroundTint">@color/yellow</item>
+    </style>
```

Comment:
both style are setting the same properties, try to use only one, with the background tint, and the text put on the layout component

Reply:
Done, changed both of the properties to layout because the style only had the background tint and would be a bad practice to leave it there alone